concepticondata/conceptlists/Greenhill-2015-2525.tsv

```diff
@@ -1187,27 +1187,27 @@ Greenhill-2015-2525-2491	2491	pipe	1124	PIPE
 Greenhill-2015-2525-2492	2492	pubic nut		
 Greenhill-2015-2525-2493	2493	pubic shell		
 Greenhill-2015-2525-2494	2494	testicle	797	TESTICLES
-Greenhill-2015-2525-2495	2495	axe (tomahawk)		
+Greenhill-2015-2525-2495	2495	axe (tomahawk)	677	AXE
 Greenhill-2015-2525-2496	2496	whiskers	1165	MOUSTACHE
 Greenhill-2015-2525-2497	2497	not know		
-Greenhill-2015-2525-2498	2498	child (f)		
-Greenhill-2015-2525-2499	2499	child (m)		
-Greenhill-2015-2525-2500	2500	pandanus (edible)		
+Greenhill-2015-2525-2498	2498	child (f)	1646	GIRL
+Greenhill-2015-2525-2499	2499	child (m)	1366	BOY
+Greenhill-2015-2525-2500	2500	pandanus (edible)	469	PANDANUS
 Greenhill-2015-2525-2501	2501	to draw water	1911	DRAW (WATER)
-Greenhill-2015-2525-2502	2502	that (distant)		
-Greenhill-2015-2525-2503	2503	there (distant)		
-Greenhill-2015-2525-2504	2504	this (close)		
-Greenhill-2015-2525-2505	2505	this (further)		
-Greenhill-2015-2525-2506	2506	this (lohap)		
-Greenhill-2015-2525-2507	2507	there (lohap)		
-Greenhill-2015-2525-2508	2508	yam (maho)		
-Greenhill-2015-2525-2509	2509	yam (taitu)		
-Greenhill-2015-2525-2510	2510	to fall (from standing)		
+Greenhill-2015-2525-2502	2502	that (distant)	78	THAT
+Greenhill-2015-2525-2503	2503	there (distant)	1937	THERE
+Greenhill-2015-2525-2504	2504	this (close)	1214	THIS
+Greenhill-2015-2525-2505	2505	this (further)	1214	THIS
+Greenhill-2015-2525-2506	2506	this (lohap)	1214	THIS
+Greenhill-2015-2525-2507	2507	there (lohap)	1937	THERE
+Greenhill-2015-2525-2508	2508	yam (maho)	410	YAM
+Greenhill-2015-2525-2509	2509	yam (taitu)	410	YAM
```

Comment:
I'm not sure about the double mapping of the glosses 2503 to 2509. Usually, we map a concept only once in these concept lists.

Reply:
ok - these are Yam cultures so there's lots of types of yam... shall I remove one or both?

Reply:
Maybe better remove both for now.